source/modules/MEI.shared.xml

```diff
@@ -4164,6 +4164,7 @@
       <memberOf key="att.accid.ges"/>
       <memberOf key="att.accid.anl"/>
       <memberOf key="model.noteModifierLike"/>
```

Comment:
This is added in ‘shared’ which is not correct.

Reply:
hi, I'm raising an issue about the accidental requirements. But where is the right place to add that if not in the shared module?

Reply:
As I understand it you want `<nc>` to allow `<accid>` as a child? Then you would add a model class to the 'content' block of `nc` that contains the elements that you want. You would probably have to create your own model class -- see [`model.noteModifierLike`](https://music-encoding.org/guidelines/v4/model-classes/model.notemodifierlike.html) as an example.

If you add it directly to shared it creates an upward dependency on the 'neume' module -- generally the specific modules should always use and refer to things in shared, but not the other way around.